### PR TITLE
Update to OPA v0.57.0

### PR DESCRIPTION
**Pull request by zregvart**
resolves: gh-1065

### CODE REVIEW

File: acceptance/go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/enterprise-contract/ec-cli/acceptance
 
-go 1.20
+go 1.21.0
+
+toolchain go1.21.1
```
Comment (lcarva) [inline acceptance/go.mod:5]: Why do we need this? Why does it have a different value than the `go` line above?

Comment (zregvart) [reply acceptance/go.mod:5]: OPA requires go 1.21

Comment (lcarva) [reply acceptance/go.mod:5]: That's not what I meant. Why do we have different values for `go` and `toolchain` ?

Comment (zregvart) [reply acceptance/go.mod:5]: There is explanation of this on [the go blog](https://go.dev/blog/toolchain#toolchain-management)

Comment (lcarva) [reply acceptance/go.mod:5]: Thanks. That blog is not super clear. I'm trying to understand. It seems to me that the value `1.21.0` could be used in both cases? Is there are reason we want the `.1` patch for toolchain?

Comment (zregvart) [reply acceptance/go.mod:5]: Just reflects what I had on the command line at the time

**Pull request merged**